src/Microsoft.PowerShell.Commands.Utility/commands/utility/Resolve-ErrorRecord.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Resolve-ErrorRecord tests.Resolve-ErrorRecord -Newest `9999` works: greater than total
```powershell
Expected 9999, but got 256.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Resolve-ErrorRecord.Tests.ps1: line 57
57:         $out.Count | Should -Be $expected
```